src/Routing/ResourceRoutingRepository.php

```diff
@@ -3,16 +3,44 @@
 namespace DoubleThreeDigital\Runway\Routing;
 
 use DoubleThreeDigital\Runway\Models\RunwayUri;
+use DoubleThreeDigital\Runway\Runway;
+use Statamic\Search\SearchResult;
 
 class ResourceRoutingRepository
 {
+    public function find($id)
+    {
+        [$resourceHandle, $id] = explode('::', $id, 2);
+
+        $resource = Runway::findResource($resourceHandle);
+        $record = $resource->model()->firstWhere($resource->primaryKey(), $id);
+
+        if (!$record) {
+            return null;
+        }
+
+        return (new SearchResult)->data([
```

Comment:
I'm a little unsure about this always returning a `SearchResult`. 

While there's no actual issue with it at the moment, it means we wouldn't be able to do a normal lookup like `Data::find('runway::product::1')` and get back the associated 'record'. Well, we could but it would give us a SearchResult, not the model.